views/templates/block/upgradeButtonBlock.twig

```diff
@@ -117,7 +117,7 @@
                                 <label for="archive_num">{{ 'Number of the version you want to upgrade to:'|trans({}, 'Modules.Autoupgrade.Admin') }} * </label>
                                 <input type="text" size="10" id="archive_num" name="archive_num" value="{{ archiveVersionNumber }}" placeholder="1.7.0.1" />
                                 <br>
-                                <label for="archive_xml">{{ 'XML file to use:'|trans({}, 'Modules.Autoupgrade.Admin') }}</label>
+                                <label for="archive_xml">{{ 'XML file to use:'|trans({}, 'Modules.Autoupgrade.Admin') }} *</label>
```

Comment:
Maybe you should add a `required` attribute.

Reply:
I haven't because I looked how it was done for other fields and it's not done, the version number L118 for example

Reply:
done (also added an asterisk to the archive label because it's mandatory in the js code)

Reply:
thank you